src/services/ReportHub/ReportHub.BLL/Extensions/GrpcRegistrationExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Grpc.Net.Client;
+using Grpc.Net.ClientFactory;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using ProtoBuf.Grpc.Client;
+using Shared.gRPC;
+
+namespace ReportHub.BLL.Extensions
+{
+    public static class GrpcRegistrationExtensions
+    {
+        public static void RegisterGrpcClient(
+            this IServiceCollection services,
+            IConfiguration config
+        )
+        {
```

Comment:
Let's use AddGrpcClient for configuration

Reply:
I use code first approach to configure gRPC and no client code is generated.
I didn't find any overloads or info how to inject client using AddGrpcClient if i don't have a client class.